### PR TITLE
fix(test): run unit tests of frontend applications

### PR DESCRIPTION
**Pull request by raphj**
This pull request fixes the frontend unit test script.
#2999 

## Checkpoints

**For code reviewer**

- [x] Code is clear enough
- [x] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [x] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation

**For developer**

- [x] Manual tests done to ensure stability on whole application layers, issue expectation follow
- [x] Original issue have been updated with a short resume of implemented solution

**For quality team**

- [x] Tested by quality team

### CODE REVIEW

**Comment by raphj** [comment]
looks good

**Comment by gbonaspetti** [comment]
Verified on branch (wait merge to close).

File: frontend_app_collaborative_document_edition/test/CollaboraEditionFrame.spec.js [hunk 1]
```diff
@@ -20,8 +20,25 @@ describe('<CollaborativeEditionFrame />', function () {
       config: {
         apiUrl: 'http://unit.test:6543/api/v2',
         translation: { en: { translation: {} } },
+        system: {
+          appListLoaded: true,
+          config: {
+            content_length_file_size_limit: 0,
+            email_notification_activated: true,
+            instance_name: "Algoo",
```
Comment (PhilippeAccorsi) [inline frontend_app_collaborative_document_edition/test/CollaboraEditionFrame.spec.js:28]: Why not choose `Tracim` in place of `Algoo` ?

Comment (PhilippeAccorsi) [inline frontend_app_collaborative_document_edition/test/CollaboraEditionFrame.spec.js:28]: Why not choose `Tracim` in place of `Algoo` ?

Comment (Skylsmoi) [reply frontend_app_collaborative_document_edition/test/CollaboraEditionFrame.spec.js:28]: mmh, it's unit test data, I didn't thought a lot about it

**Pull request merged**